fixtures/github.py

```diff
@@ -3541,3 +3541,16 @@
     "site_admin": false
   }
 }"""
+
+# Simplified example of a check_run rerequested action event
```

Comment:
are there docs that list the event format if we want to check the full payload later?